core/Enclave/enclave_hsm.cpp

```diff
@@ -338,23 +477,23 @@ sgx_status_t enclave_aes_decrypt(const uint8_t *aad, size_t aad_len,
     uint8_t *iv = (uint8_t *)(cipherblob + plaintext_len);
     uint8_t *mac = (uint8_t *)(cipherblob + plaintext_len + EH_AES_GCM_IV_SIZE );
 
-    sgx_key_128bit_t dec_key;
+    uint8_t * enc_key = new uint8_t[key_size];
```

Comment:
use malloc/free but not new/delete. I remember it's the requirement from crypto review. @syan10

Reply:
since key_size is small, the stack can be used as below: 
uint8_t enc_key[key_size] = {0};

Reply:
I'd like to use 'malloc->memset_s->free' for it.